native/general.go

```diff
@@ -4,20 +4,57 @@
 
 package native
 
-import "github.com/gonum/lapack"
+import (
+	"math"
 
+	"github.com/gonum/lapack"
+)
+
+// Implementation is the native Go implementation of LAPACK routines. It
+// is built on top of calls to the return of blas64.Implementation, so while
```

Comment:
Yes. I added parentheses, but it's the type returned by http://godoc.org/github.com/gonum/blas/blas64#Implementation